test/ao/sparsity/test_data_sparsifier.py

```diff
@@ -230,3 +232,209 @@ def test_nn_embeddings(self):
         emb_test = TestBaseDataSparsiferType(data_list=data_list, defaults=defaults,
                                              data_with_config=data_with_config)
         emb_test.run_tests()
+
+
+class TestNormDataSparsifierType(TestBaseDataSparsiferType):
+    def __init__(self, data_list, defaults, data_with_config, norm_type='L1'):
+        super().__init__(data_list=data_list, defaults=defaults, data_with_config=data_with_config)
+        assert norm_type in ['L1']
+        self.norm_type = norm_type
+
+    def get_bounds_on_actual_sparsity(self, config, tensor_shape):
```

Comment:
if these are helper classes for other tests, should probably prepend `_` to signify it being private.